Resources/ModelingAndBinding-Starter/Podfile

```diff
@@ -0,0 +1,11 @@
+# Uncomment the next line to define a global platform for your project
+platform :ios, '10.0'
+
+use_frameworks!
+
+pod 'IGListKit', :git => 'https://github.com/Instagram/IGListKit.git', :branch => 'master'
```

Comment:
Ahh. I see. Maybe we keep the markdown guide here and all the code, etc. somewhere else?

Starting to feel a bit bloated in this repo. Could we put the projects under `rnystrom/IGListKit-binding-guide` or something?